static/app/views/detectors/components/details/uptime/index.tsx

```diff
@@ -19,12 +21,24 @@ type UptimeDetectorDetailsProps = {
 export function UptimeDetectorDetails({detector, project}: UptimeDetectorDetailsProps) {
   const dataSource = detector.dataSources[0];
 
+  const {selection} = usePageFilters();
```

Comment:
It would be nice if we could avoid adding this to every new detector type. Could we put this in `DetectorDetailsOngoingIssues`? If we need to override it for metric detectors you set it up so that when it receives a prop it would take precedence.